fork/dynamic-cdn-webpack-plugin/src/index.js

```diff
@@ -276,7 +276,9 @@ class DynamicCdnWebpackPlugin {
 
 	async addModule(contextPath, modulePath, { env, isOptional = false }) {
 		const isModuleExcluded =
-			this.exclude.includes(modulePath) || (this.only && !this.only.includes(modulePath));
+			this.exclude.includes(modulePath) ||
+			(this.only && !this.only.includes(modulePath)) ||
+			modulePath.startsWith('@types/');
```

Comment:
handle
DynamicCdnWebpackPlugin ERROR: 
❌ @types/d3-interpolate couldn't be loaded because it is not found by require.resolve

DynamicCdnWebpackPlugin ERROR: 
❌ @types/d3-scale couldn't be loaded because it is not found by require.resolve

DynamicCdnWebpackPlugin ERROR: 
❌ @types/d3-shape couldn't be loaded because it is not found by require.resolve
waiting for https://github.com/recharts/recharts/pull/2843